src/Package/Impl/DataInspect/VariableGridWindowPane.cs

```diff
@@ -24,5 +24,10 @@ internal sealed class VariableGridWindowPane : RToolWindowPane {
             }
             _gridHost.SetEvaluation(evaluation);
         }
+
+        protected override void OnClose() {
+            base.OnClose();
+            _gridHost.CleanUp();
```

Comment:
I verified that, detaching details view tab, toggling between tabs, and layout changes when debugging starts does not call OnClose.